gm_global_a_powertrain.dbc

```diff
@@ -147,6 +147,20 @@ BO_ 489 EBCMVehicleDynamic: 8 K17_EBCM
  SG_ YawRate : 35|12@0- (0.625,0) [0|1] "" NEO
  SG_ YawRate2 : 51|12@0- (0.0625,0) [-2047|2047] "grad/s" NEO
 
+BO_ 512 GAS_COMMAND: 6 NEO
```

Comment:
these should go in their own DBC that gets included like the honda and toyota

Reply:
That is a sizeable change. I will remove these from this PR

Reply:
Removed.